package-lock.json

```diff
@@ -1299,7 +1299,10 @@
       "requires": {
         "@lerna/query-graph": "file:utils/query-graph",
         "figgy-pudding": "^3.5.1",
-        "p-queue": "^4.0.0"
+        "fs-extra": "^8.1.0",
+        "npmlog": "^4.1.2",
+        "p-queue": "^6.2.1",
+        "upath": "^1.2.0"
```

Comment:
What does `upath` provide that the builtin `path` doesn't? Cross platform normalization is available via `path.normalize()`.

Reply:
If `profileLocation` is provided with `\\` as separators, and lerna is run in a posix environment, or if `profileLocation` is provided with `/` as separators, and lerna is run in a Windows environment, there will be a mismatch path separator mismatch, as far as I can see. And that will result in runtime errors.

```
> upath.join("c:\\windows\\nodejs\\path", "packages/foo")
"c:/windows/nodejs/path/packages/foo"

> path.join("c:\\windows\\nodejs\\path", "packages/foo")
"c:\\windows\\nodejs\\path/packages/foo"

> upath.join("/Users/root/nodejs", "packages\\foo")
"/Users/root/nodejs/packages/foo"

> path.join("/Users/root/nodejs", "packages\\foo")
"/Users/root/nodejs/packages\\foo"

> path.normalize(path.join("/Users/root/nodejs", "packages\\foo"))
"/Users/root/nodejs/packages\\foo"
```

Reply:
I looked into it, `upath` seems pretty solid. Thanks for introducing me to the package. :)